server/graphql/v2/object/Collective.js

```diff
@@ -9,7 +9,7 @@ export const Collective = new GraphQLObjectType({
   name: 'Collective',
   description: 'This represents a Collective account',
   interfaces: () => [Account, AccountWithHost, AccountWithContributions],
-  isTypeOf: collective => collective.type === collectiveTypes.COLLECTIVE,
+  isTypeOf: collective => collective.type === collectiveTypes.COLLECTIVE || collectiveTypes.ORGANIZATION,
```

Comment:
In GraphQL V2, an `Organization` is not a `Collective`, these are two different types. Why is that needed?

Reply:
Thanks for the clarification, I thought Organizaton is a collective. Let me fix it